eng/common/TestResources/deploy-stress-test-resources.ps1

```diff
@@ -0,0 +1,40 @@
+param (
```

Comment:
Are you planning on moving this file back into `tools/stress-cluster` somewhere now that we've changed the approach?